hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/source/IncrementalInputSplits.java

```diff
@@ -135,44 +127,37 @@ public static Builder builder() {
   public Result inputSplits(
       HoodieTableMetaClient metaClient,
       boolean cdcEnabled) {
-    HoodieTimeline commitTimeline = getReadTimeline(metaClient);
-    if (commitTimeline.empty()) {
-      LOG.warn("No splits found for the table under path " + path);
+
+    IncrementalQueryAnalyzer analyzer = IncrementalQueryAnalyzer.builder()
+        .metaClient(metaClient)
+        .startTime(this.conf.getString(FlinkOptions.READ_START_COMMIT))
+        .endTime(this.conf.getString(FlinkOptions.READ_END_COMMIT))
```

Comment:
When is it possible that both READ_START_COMMIT and READ_END_COMMIT are null/empty?

Reply:
In streaming query, the user may declares nothing. And similar with Kafka, it consumes from the latest commit.